### ISSUE TITLE
Unable to launch any game with proton 5.0-7

### ISSUE THREAD

**Issue by astiak** [ValveSoftware/Proton#3879]
Attempting to debug this for days, but I've tried everything I could think of.

I'm unable to launch any game with the latest version of proton (5.0-7). I'm fairly certain this has been ongoing for me since proton 5.0-6, where 5.0-5 was the last version to work for me. For example, when I launch "THE WALKING DEAD" it refuses to launch with proton 5.0-7. If I use proton 4.11-13, I have no issues launching the game. This is also the case for "Vanquish" and "Dragon Age: Origins".

Prior to this I was able to play most of my games on one of the branches of 5.0 but I can't remember for certain which one was the last working one for me (I have a feeling it was 5.0-5).

Attached are the proton log files for the attempt at running these games with 5.0-7
[steam-17450.log](https://github.com/ValveSoftware/Proton/files/4638760/steam-17450.log)
[steam-207610.log](https://github.com/ValveSoftware/Proton/files/4638761/steam-207610.log)
[steam-460810.log](https://github.com/ValveSoftware/Proton/files/4638762/steam-460810.log)

I use an NVIDIA prime setup, so I launch my games using a script to use the NVIDIA card while steam runs on the integrated graphics.

What I've tried:
- Resetting launch options.
- Remove the Proton 5.0 directory and allow it to be re-downloaded
- Completely uninstalled steam and removed all steam related directories & files
- Launching games on both integrated and discrete cards

In all cases above, the games fail to run on 5.0, but run fine on 4.11.

I've provided the Steam system information for steam running on both Intel and NVIDIA.

[sys-info-intel.txt](https://github.com/ValveSoftware/Proton/files/4638780/sys-info-intel.txt)
[sys-info-nvidia.txt](https://github.com/ValveSoftware/Proton/files/4638781/sys-info-nvidia.txt)

Please do let me know if any further information is required.

**Comment by astiak** [comment]
I should add, I am not using NTFS. All my drives are using ext4 filesystems.

**Comment by kisak-valve** [comment]
Hello @astiak, this looks like a line of interest in the logs:
```
terminate called after throwing an instance of 'Json::LogicError'
  what():  in Json::Value::find(key, end, found): requires objectValue or nullValue
```

**Comment by Newbytee** [comment]
Could this be relevant: https://github.com/ValveSoftware/Proton/issues/607#issuecomment-629613054

**Comment by aeikum** [comment]
Hey @astiak thanks for reporting. Can you check for a file at `$HOME/.config/openvr/openvrpaths.vrpath`? Does it exist and have contents? What are those contents? (Alternatively, if you have a custom `XDG_CONFIG_HOME` set, this file will be at `$XDG_CONFIG_HOME/openvr/openvrpaths.vrpath`.)

**Comment by astiak** [comment]
Hi @aeikum, thanks for looking into this. The file does exist and the contents are as follows:
`"config" : [ "/home/astiak/.local/share/Steam/config/" ], "external_drivers" : null, "log" : [ "/home/astiak/.local/share/Steam/logs/" ], "runtime" : [ 
"/home/astiak/.local/share/Steam/SteamApps/common/SteamVR" ] }`

**Comment by aeikum** [comment]
Is it really missing the first open brace `{` ?

**Comment by astiak** [comment]
Looks like it, yes! Just double checked to make sure I didn't mess up the copy-pasting of the previous comment.

Added a .txt extension to the file so I could attach to this comment.
[openvrpaths.vrpath.txt](https://github.com/ValveSoftware/Proton/files/4646584/openvrpaths.vrpath.txt)

**Comment by astiak** [comment]
Well @aeikum that seems to be it! Just tested two games, and both launched with 5.0-7 with no issues.

Thanks very much for the guidance and help! I'm not quite sure how this brace went missing and how it didn't affect older proton versions.

It now shows this:
`{ "config" : [ "/home/astiak/.local/share/Steam/config/" ], "external_drivers" : null, "log" : [ "/home/astiak/.local/share/Steam/logs/" ], "runtime" : [ 
"/home/astiak/.local/share/Steam/SteamApps/common/SteamVR" ] }`

**Comment by aeikum** [comment]
Great, glad we figured it out. In any case, an invalid vrpaths file shouldn't kill Proton, so I'll look into fixing that. Thanks for reporting the issue!

**Comment by aeikum** [comment]
This was fixed in Proton 5.0-9 with ee8e0653676568eb3b3c2a973a3b1723d094f537. Thanks for reporting and helping to figure out the problem!

**Comment by astiak** [comment]
Thanks @aeikum for the permanent fix.
